### PR TITLE
[AWS-3495] f1.4xlarge instance types

### PR DESCRIPTION
**Pull request by NatahlieBeavers**
#### Reviewers! 
Here's [How to do a Code Review](https://cloudhealthtech.atlassian.net/wiki/spaces/EN/pages/17989636/Code+Review)

#### Deploy readiness:

- [ ]  2 Reviews: 1 peer/buddy and 1 lieutenant/SME from [here](https://cloudhealthtech.atlassian.net/wiki/spaces/EN/pages/17989636/Code+Review)
- [ ]  Did you add the JIRA ticket ID to the PR title? 
- [ ]  Is your PR title **meaningful?** `[ENG-3333] Update pull request template`, not `Typo Fix` or `Add test`
- [ ]  After commits are squashed, will your primary commit message be meaningful?

#### Documentation links:

- JIRA Ticket(s): https://cloudhealthtech.atlassian.net/browse/AWS-3495
- Design Doc (where applicable):
- Link to passing Solano build (if PR tests are failing due to inter-repo dependencies):
  - [ ]  Branch pointers in Gemfile removed?

#### Proposed changes:
- Add support for f1.4xlarge
-

#### Test Plan and Risk Assessment:
_Will this change impact other teams' code? If so, how?_

-Create instance types f1.4xlarge

_Test Plan:_

- Confirm that after doing price collection and update on local machine, that records are inserted to the db for prices for new instances

_Rollback Plan:_

- revert, rebuild cubes

### CODE REVIEW

File: lib/amazon-pricing/definitions/instance-type.rb [hunk 1]
```diff
@@ -319,6 +319,7 @@ def self.get_values(json, category_type, override_price = false)
       'x1e.xlarge' => 64, 'x1e.2xlarge' => 64, 'x1e.4xlarge' => 64, 'x1e.8xlarge' => 64, 'x1e.16xlarge' => 64, 'x1e.32xlarge' => 64,
       'p2.xlarge' => 64, 'p2.8xlarge' => 64, 'p2.16xlarge' => 64,
       'p3.2xlarge' => 64, 'p3.8xlarge' => 64, 'p3.16xlarge' => 64,
+      'f1.2xlarge' => 64, 'f1.4xlarge' => 64, 'f1.16xlarge' => 64,
```
Comment (jeffzhou791) [inline lib/amazon-pricing/definitions/instance-type.rb]: did we not have f1 at all here?

Comment (NatahlieBeavers) [reply lib/amazon-pricing/definitions/instance-type.rb]: We did not. Not sure why.

Comment (stanmluke) [reply lib/amazon-pricing/definitions/instance-type.rb]: are we sure of the CPU `bit width` here?  remember the f1 is a FPGA, and as I understand it is technically a horizontal (could be 256b wide) cpu?

**Comment by NatahlieBeavers** [comment]
I did see it on this page: https://aws.amazon.com/ec2/dedicated-hosts/pricing/. Will add to the @@Capacity_hash

**Pull request merged**